test/services/policymanager/serverpolicymanager.go

```diff
@@ -4,33 +4,62 @@
 package main
 
 import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
 	"log"
-	"net"
+	"net/http"
+	"strconv"
 
 	mockup "fybrik.io/fybrik/manager/controllers/mockup"
-	"fybrik.io/fybrik/manager/controllers/utils"
-	pb "fybrik.io/fybrik/pkg/connectors/protobuf"
-	"google.golang.org/grpc"
+	openapiclientmodels "fybrik.io/fybrik/pkg/taxonomy/model/base"
+	"github.com/gin-gonic/gin"
 )
 
 const (
 	PORT = 50082
 )
 
-func main() {
-	address := utils.ListeningAddress(PORT)
-	log.Printf("starting mock policy manager server on address %s", address)
+var router *gin.Engine
 
-	listener, err := net.Listen("tcp", address)
+func constructPolicyManagerRequest(inputString string) *openapiclientmodels.PolicyManagerRequest {
```

Comment:
fmt? where does it go? is this the right output mechanism?

Reply:
changed to use log package. Thanks.